### ISSUE TITLE
CIFAR10DVS复现精度低1个点

### ISSUE THREAD

**Issue by Bobby-xb** [zhouchenlin2096/Spikingformer#12]
作者您好，我在实际跑spikingformer的CIFAR10DVS时，复现精度为80.3，论文精度汇报为81.3。

这是我的部分训练日志，没有修改默认的设置：
Namespace(model='SEWResNet', dataset='cifar10dvs', num_classes=10, data_path='', device='cuda', batch_size=16, workers=4, print_freq=256, output_dir='./logs', resume='', sync_bn=False, test_only=False, amp=True, world_size=1, dist_url='env://', tb=True, T=16, opt='adamw', opt_eps=1e-08, opt_betas=None, weight_decay=0.06, momentum=0.9, connect_f='ADD', T_train=None, sched='cosine', lr=0.001, lr_noise=None, lr_noise_pct=0.67, lr_noise_std=1.0, lr_cycle_mul=1.0, lr_cycle_limit=1, warmup_lr=1e-05, min_lr=1e-05, epochs=96, epoch_repeats=0.0, start_epoch=0, decay_epochs=20, warmup_epochs=10, cooldown_epochs=10, patience_epochs=10, decay_rate=0.1, smoothing=0.1, mixup=0.5, cutmix=0.0, cutmix_minmax=None, mixup_prob=0.5, mixup_switch_prob=0.5, mixup_mode='batch', mixup_off_epoch=0, distributed=False)
Loading data
The directory [/dataset/CIFAR10DVS/frames_number_16_split_by_number] already exists.
Took 68.00592994689941
Creating data loaders
Creating model
number of params: 2568714
purge_step_train=0, purge_step_te=0
Start training
......
Epoch: [105]  [  0/562]  eta: 0:07:32  lr: 1e-05  img/s: 47.72630481488409  loss: 0.5274 (0.5274)  acc1: 100.0000 (100.0000)  acc5: 100.0000 (100.0000)  time: 0.8052  data: 0.4699  max mem: 4091
Epoch: [105]  [256/562]  eta: 0:01:25  lr: 1e-05  img/s: 58.787906183040135  loss: 0.5334 (0.5567)  acc1: 100.0000 (98.9056)  acc5: 100.0000 (100.0000)  time: 0.2857  data: 0.0002  max mem: 4091
Epoch: [105]  [512/562]  eta: 0:00:13  lr: 1e-05  img/s: 52.48202004845539  loss: 0.5374 (0.5547)  acc1: 100.0000 (99.1228)  acc5: 100.0000 (99.9756)  time: 0.2597  data: 0.0002  max mem: 4091
Epoch: [105] Total time: 0:02:37
Test:  [ 0/63]  eta: 0:00:33  loss: 0.9925 (0.9925)  acc1: 68.7500 (68.7500)  acc5: 93.7500 (93.7500)  time: 0.5348  data: 0.4309  max mem: 4091
Test: Total time: 0:00:06
Acc@1 = 79.9, Acc@5 = 97.3, loss = 0.7041220109141062
Namespace(model='SEWResNet', dataset='cifar10dvs', num_classes=10, data_path='/dataset/CIFAR10DVS', device='cuda', batch_size=16, workers=4, print_freq=256, output_dir='./logs', resume='', sync_bn=False, test_only=False, amp=True, world_size=1, dist_url='env://', tb=True, T=16, opt='adamw', opt_eps=1e-08, opt_betas=None, weight_decay=0.06, momentum=0.9, connect_f='ADD', T_train=None, sched='cosine', lr=0.001, lr_noise=None, lr_noise_pct=0.67, lr_noise_std=1.0, lr_cycle_mul=1.0, lr_cycle_limit=1, warmup_lr=1e-05, min_lr=1e-05, epochs=96, epoch_repeats=0.0, start_epoch=0, decay_epochs=20, warmup_epochs=10, cooldown_epochs=10, patience_epochs=10, decay_rate=0.1, smoothing=0.1, mixup=0.5, cutmix=0.0, cutmix_minmax=None, mixup_prob=0.5, mixup_switch_prob=0.5, mixup_mode='batch', mixup_off_epoch=0, distributed=False)
Training time 4:36:10 max_test_acc1 **80.3** test_acc5_at_max_test_acc1 97.3
./logs/SEWResNet_b16_T16_wd0.06_adamw_cnf_ADD/lr0.001

请问想要复原81.3的精度需要额外修改什么配置吗？感谢您的回复。
